llvm_jni/modules/jni.llvm/Module.bal

```diff
@@ -7,9 +7,19 @@ function intrinsicNameToId(IntrinsicFunctionName name) returns int {
     return jLLVMLookupIntrinsicID(java:fromString(str_name), str_name.length());
 }
 
+// JBUG: #31447 make possible values for these strict
+public type LLVMCodeGenOptLevel string;
+public type LLVMRelocMode string;
+public type LLVMCodeModel string;
+
+configurable string optLevel = "Default";
```

Comment:
I don't think something like optLevel should be handled via configurable. This is something that should be a command-line option.